src/main/java/duke/Exceptions/descriptionException.java

```diff
@@ -0,0 +1,11 @@
+package duke.Exceptions;
+public class descriptionException extends Exception {
```

Comment:
Perhaps you could add a line between these two lines. I noticed this issue in other places as well.